.travis.yml

```diff
@@ -24,6 +24,7 @@ matrix:
       dist: trusty
       sudo: required
       services: docker
+  fast_finish: true
```

Comment:
fantastically simple!
Is it normal that the indenting is different?

Reply:
It's indented only one level (two spaces) below `matrix:` because it's a setting on matrix, not a specific OS in `include:`